rector.php

```diff
@@ -56,5 +56,5 @@
             __DIR__ . '/src/Console/Notifier.php',
         ],
 
-        RemoveUnusedPrivatePropertyRector::class => [__DIR__ . '/src/Configuration/RectorConfigBuilder.php'],
+        RemoveUnusedPrivatePropertyRector::class => [__DIR__ . '/src/Configuration/RectorConfigBuilder.php']
```

Comment:
The original trailing comma `,` is from ecs, should be readded, as it will be re-add later by weekly PR for coding standard apply

Reply:
Argh, sorry, added again